scripts/spec/cli_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'spec_helper'
+require 'cli'
+
+describe Build::Cli do
+  context "#parse" do
+    it "only (default)" do
+      expect(described_class.new.parse(%w()).options[:only]).to match_array %w(vsphere ovirt openstack)
+    end
+
+    it "only vsphere and ovirt" do
+      expect(described_class.new.parse(%w(-o vsphere ovirt)).options[:only]).to match_array %w(vsphere ovirt)
```

Comment:
ok, the difference between `:multi => true` and `:type ==> :strings`. confusing. Who manages this darn trollop gem anyway?